### ISSUE TITLE
HLint suggest to rewrite RULES

### ISSUE THREAD

**Issue by phadej** [ndmitchell/hlint#633]
Given `File.hs`

```haskell
newtype Id a = Id a

instance Functor Id where
  fmap f (Id x) = Id (f x)

{-# RULES "Id-fmap-id" forall (x :: Id a). fmap id x = x #-}
```

HLint suggests

```
File.hs:6:44: Warning: Functor law
Found:
  fmap id
Perhaps:
  id
```
---

```
% hlint --version
HLint v2.1.17, (C) Neil Mitchell 2006-2019
```

**Comment by ndmitchell** [comment]
Thanks, good point - fixed.
